File: go/srv.go
package main

import (
	"io"
	"log"
	"net/http"
)

func handler(w http.ResponseWriter, r *http.Request) {
	w.Header().Set("Connection", "keep-alive")
	w.Header().Set("Content-Type", "text/plain")
	w.Header().Set("Content-Length", "5")
	w.WriteHeader(http.StatusOK)
	io.WriteString(w, "Hello")
}

func main() {
	http.HandleFunc("/", handler)
	log.Fatal(http.ListenAndServe(":7012", nil))
}

File: python/srv.py
from http.server import BaseHTTPRequestHandler, HTTPServer

class Handler(BaseHTTPRequestHandler):
    def do_GET(self):
        self.send_response(200)
        self.send_header("Connection", "keep-alive")
        self.send_header("Content-type", "text/plain")
        self.send_header("Content-length", "5")
        self.end_headers()
        self.wfile.write(bytes("Hello", "utf-8"))
    def log_message(self, format, *args):
        return

server = HTTPServer(("", 7012), Handler)

try:
    server.serve_forever()
except KeyboardInterrupt:
    pass

server.server_close()
